manager/repository/transient/transient.go

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
Yes. We have both in service. Manager falls back the transient repository if the persistent one is not available.